labels.md

```diff
@@ -0,0 +1,219 @@
+priority/non-release-blocker    #0052cc     P: High                             #eb6420
+priority/release-blocker        #b60205     P: Blocker                          #eb6420
+proposal                        #eb6420     T: Proposal                         #1d76db
+question                        #fbca04     T: Question                         #1d76db
+release-note                    #1d76db     * <T: CHANGELOG>                    
+require-LGT3                    #d93f0b     * <S: Waiting on review>            
+status/DNM                      #b60205     S: DNM                              #e6e6e6
+status/LGT1                     #d4c5f9     * <S: Waiting on review>            
+status/LGT2                     #5319e7     * <S: Waiting on review>            
+status/LGT3                     #330099     * <S: Waiting on review>            
+status/PTAL                     #2cbe4e     S: Waiting on review                #e6e6e6
+status/ReqChange                #b60205     S: Waiting on author                #e6e6e6
+status/TODO                     #207de5     *                                   
+status/WIP                      #fbca04     S: WIP                              #e6e6e6
+status/all tests passed         #2cbe4e     * <S: Bot Merge>                    
+status/can merge                #2cbe4e     S: Bot Merge                        #e6e6e6
+status/future                   #fbca04     ?                                   
+type/1.0 cherry-pick            #c2e0c6     *                                   
+type/2.0 cherry-pick            #99d0ef     *                                   
+type/2.1 cherry-pick            #9bfff5     * <T: CherryPick-2.1>               
+type/3.0                        #ededed     *                                   
+type/3.0 cherry-pick            #297b99     * <T: CherryPick-3.0>               
+type/TEP                        #bfd4f2     T: TEP                              #1d76db
+type/bug                        #fc2929     T: Bug                              #1d76db
+type/bug-fix                    #fc2929     T: BugFix                           #1d76db
+type/compatibility              #e99695     T: Compatibility                    #1d76db
+type/duplicate                  #cccccc     S: Closed dup                       #e6e6e6
+type/enhancement                #84b6eb     T: Enhancement                      #1d76db
+type/invalid                    #e6e6e6     S: Closed invalid                   #e6e6e6
+type/investigation              #f46100     T: Investigation                    #1d76db
+type/new-feature                #e2e876     * <T: Enhancement>
+type/performance                #006b75     T: Performance                      #1d76db
+type/regression                 #c41b2f     T: Regression                       #1d76db
+type/suggestion                 #11b2c4     * <T: Proposal>                     
+type/wontfix                    #95d3db     * <just close the issue>            
```

Comment:
Maybe `s: Closed WontFix` is better.

Reply:
see https://github.com/pingcap/tidb/pull/11729#discussion_r313598554

Reply:
see https://github.com/pingcap/tidb/pull/11729#discussion_r313598554